assignment01/Exercise1.java

```diff
@@ -0,0 +1,13 @@
+package assignment01;
+
+public class Exercise1 {
+    public int largestNumber(int number) {
+        return (int) Math.pow(10, number) - 1;
+    }
+
+    public static void main(String[] args) {
+        System.out.println("------------- start -------------");
+        System.out.println(" largest to 2 = " + new Exercise1().largestNumber(2));

```

Comment:
Awesome! `new Exercise1().largestNumber(2)` is totally correct but you will improve the performance if you convert your method to be static `public static int largestNumber(int number) `, that way you can call the method directly here.